tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,21 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func getVcpuMask(pod *k8sv1.Pod, emulator, cpu string) (output string, err error) {
+	virtClient := kubevirt.Client()
+
+	pscmd := `ps -LC ` + emulator + ` -o lwp,comm | grep "CPU ` + cpu + `"  | cut -f1 -dC`
+	args := []string{"/bin/bash", "-c", pscmd}
+	Eventually(func() error {
+		output, err = exec.ExecuteCommandOnPod(virtClient, pod, "compute", args)
+		return err
+	}).Should(Succeed())
+	vcpupid := strings.TrimSpace(strings.Trim(output, "\n"))
+	tasksetcmd := "taskset -c -p " + vcpupid + " | cut -f2 -d:"
+	args = []string{"/bin/bash", "-c", tasksetcmd}
+	output, err = exec.ExecuteCommandOnPod(virtClient, pod, "compute", args)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
The move itself if fine, but there seems to be other things that merits cleaning in the function. Please consider fixing those in another commit or PR as well.